### ISSUE TITLE
In an ASP.NET Core 3.1 API, the tests are always passed

### ISSUE THREAD

**Issue by anaselhajjaji** [BenMorris/NetArchTest#47]
My test is:

```cs
public void ApiController_DoesNotHaveDependency_ToRepository()
{
    Assembly ApiAssembly = typeof(Startup).Assembly;

    var otherModules = new List<string>
    {
        "ModuleNamespace"
    };
    var result = Types.InAssembly(ApiAssembly)
        .That()
        .ResideInNamespace("TheApiNamespace")
        .Should()
        .NotHaveDependencyOnAny(otherModules.ToArray())
        .GetResult();

    Assert.That(result.FailingTypes, Is.Null.Or.Empty);
}
```

When I tried to figure out why I noticed the following thing:
![image](https://user-images.githubusercontent.com/3309485/76236749-87bdac80-6203-11ea-83d5-7180a8341c8c.png)

It's like the library doesn't see the types in the assembly.

**Comment by BenMorris** [comment]
Sorry, but I just can't replicate this.

I'd need to see an example of a class that is not being picked up?

**Comment by anaselhajjaji** [comment]
Thank you for your prompt answer.
My project is in: https://github.com/anaselhajjaji/systemdhealthcheck
The ArchTest in: https://github.com/anaselhajjaji/systemdhealthcheck/tree/master/ArchTests

**Comment by NeVeSpl** [comment]
Problem is in that every type inside of namespace that starts with prefix "System" is excluded, and your project namespace is "SystemdHealthcheck". Fixed in #48.

**Comment by anaselhajjaji** [comment]
Yes, it was due to the System word in the namespace, I renamed my namespaces and it worked.
Thank you, I'll close the issue.
